controller/test-files/work_item_type/list/ok_agile.res.payload.golden.json

```diff
@@ -173,16 +173,14 @@
               "baseType": "string",
               "kind": "enum",
               "values": [
+                "None",
                 "Done",
-                "Rejected",
                 "Duplicate",
                 "Incomplete Description",
                 "Can not Reproduce",
-                "Partially Completed",
```

Comment:
Why can't an impediment be partially complete? I mean if the impediment is to buy a new server for example, then you could argue that placing the order and installing it are two broad steps on resolving the impediment, no?

Reply:
Steve and George did some work on this list, so I'd say this is good. We have a lot of values to choose from there and if someone later needs a special usecase, this can be added using the customization.